integration/hsm/helpers.go

```diff
@@ -244,13 +240,14 @@ func newAuthConfig(t *testing.T, log utils.Logger) *servicecfg.Config {
 
 	config.Auth.Enabled = true
 	config.Auth.NoAudit = true
-	config.Auth.ListenAddr.Addr = net.JoinHostPort(hostName, "0")
+	config.Auth.ListenAddr.Addr = net.JoinHostPort("localhost", "0")
```

Comment:
not sure why, but I had to make this change to get the tests to pass on my macbook